keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -422,28 +413,84 @@ required) or even code snippets. If there's any ambiguity about HOW your
 
 That being said, for less stringent identifiers, for example a user-specified and human-readable value, a given `id.k8s.io ClusterClaim` may need to change if an identical identifier is in use by another member of the ClusterSet it wants to join. It is likely this would need to happen outside the cluster-local boundary; for example, whatever manages memberships would likely need to deny the incoming cluster, and potentially assign (or prompt the cluster to assign itself) a new ID.
 
+Since this KEP does not formally mandate that the cluster ID *must* be immutable for the lifetime of the cluster, only for the lifetime of its membership in a ClusterSet, any dependent tooling explicitly *cannot* assume the `id.k8s.io ClusterClaim` for a given cluster will stay constant on its own merit. For example, log aggregation of a given cluster ID based on this claim should only be trusted to be referring to the same cluster for as long as it has one ClusterSet membership; similarly, controllers whose logic depends on distinguishing clusters by cluster ID can only trust this claim to disambiguate the same cluster for as long as the cluster has one ClusterSet membership.
+
+Despite this flexibility in the KEP, clusterIDs may still be useful before ClusterSet membership needs to be established; again, particularly if the implementation chooses the broadest restrictions regarding immutability and uniqueness. Therefore, having a controller that initializes it early in the lifecycle of the cluster, and possibly as part of cluster creation, may be a useful place to implement it, though within the bounds of this KEP that is not strictly necessary.
+
+The most common discussion point within the SIG regarding whether an implementation should favor a UUID or a human-readable clusterID string is when it comes to DNS. Since DNS names are originally intended to be the a human readable technique of address, clunky DNS names composed from long UUIDs seems like an anti-pattern, or at least unfinished. While some extensions to this spec have been discussed as ways to leverage the best parts of both (ex. using labels on the `id.k8s.io ClusterClaim` to store aliases for DNS), an actual API specification to allow for this is outside the scope of this KEP at this time (see the Non-Goals section).
+
 ```
-<<[UNRESOLVED]>>
-Effect of different identifier styles (mainly UUID vs human readable) on DNS
-<<[/UNRESOLVED]>>
+# An example object of `id.k8s.io ClusterClaim`:
+
+apiVersion: multicluster.k8s.io/v1
+kind: ClusterClaim
+metadata:
+  name: id.k8s.io
+spec:
+  value: 721ab723-13bc-11e5-aec2-42010af0021e
```

Comment:
Were you going to add a connection to DNS here?

Reply:
The diff makes it look like this example yaml is resolving that comment, but where I consider to have covered it (possibly not throughly enough) in the paragraph of line 435 just above (https://github.com/kubernetes/enhancements/pull/2408/files#diff-8bc62b436dee634db23bfcfd2388e6a38ac9edff8c481e6347f873765a7bae7fR435) -- which basically just says "we hear you but we're not suggesting anything yet that would make this easy". That being said I do think it's valid to to put another example yaml here where the spec.value is **not** a UUID to at least show concretely that you *could*.

Reply:
+1 to that, multiple examples to show flexibility seems like a good approach

Reply:
Added in 870f393ef201aa596d3170cc4f0446247618331c